unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/Scripts/Common/EntityInformationController.cs

```diff
@@ -86,7 +87,11 @@ public void Dispose()
 
     public void PositionChanged(Vector3 pos) { OnPositionChange?.Invoke(pos); }
 
-    public void RotationChanged(Vector3 rot) { OnRotationChange?.Invoke(rot); }
+    public void RotationChanged(Vector3 rot)
+    {
+        currentEntity?.SetRotation(rot);
+        OnRotationChange?.Invoke(rot);
```

Comment:
Please, update the `RotationChangedCorrectly` test in `EntityInformationControllerShould` taking on account this new line added.